airflow/config_templates/config.yml

```
@@ -337,7 +337,7 @@
       default: "{AIRFLOW_HOME}/logs"
     - name: remote_logging
       description: |
-        Airflow can store logs remotely in AWS S3, Google Cloud Storage or Elastic Search.
+        Airflow can store logs remotely in AWS S3, Azure Blob Storage, Google Cloud Storage or Elasticsearch.
```

```suggestion
        Airflow can store logs remotely in remote sources
```
We have too long list.